include/z64item.h

```diff
@@ -91,6 +91,9 @@ typedef enum {
     /* 0x1E */ GI_ARROWS_SMALL,
     /* 0x1F */ GI_ARROWS_MEDIUM,
     /* 0x20 */ GI_ARROWS_LARGE,
+    /* 0x22 */ GI_QUIVER_30 = 0x22,
```

Comment:
```suggestion
    /* 0x1E */ GI_ARROWS_10,
    /* 0x1F */ GI_ARROWS_30,
    /* 0x20 */ GI_ARROWS_40,
    /* 0x21 */ GI_ARROWS_50,
    /* 0x22 */ GI_QUIVER_30,
```
This looks like the missing piece here